ui/src/app/index.tsx

```diff
@@ -1,5 +1,7 @@
 import * as React from 'react';
-import * as ReactDOM from 'react-dom';
 import {App} from './app';
+import {createRoot} from 'react-dom/client';
```

Comment:
we should leave the import in the same place; the `react` deps and the external deps were grouped and alphabetized previously.

```suggestion
import {createRoot} from 'react-dom/client';

import {App} from './app';
```

I've been meaning to add an ESLint or Prettier plugin to auto sort imports for a bit now

Reply:
Imports sorted in #13813